templates/hardware.html

```diff
@@ -0,0 +1,153 @@
+{% extends '_layout.html' %} {% block title%} Ubuntu Desktop certified hardware
+{% endblock %} {% block content %}
+
+{{ details.release_details }}
+
+<nav role="navigation" class="nav-secondary clearfix">
+  <ul class="breadcrumb">
+    <li>
+      <a href="/desktop/">
+        Ubuntu Desktop
+      </a>
+    </li>
+    <li>&nbsp;›</li>
+
+    <li>
+      <a href="https://certification.ubuntu.com/desktop/models/?category=Desktop&amp;category=Laptop&amp;vendors=Dell">Search
```

Comment:
Another weirdly placed newline

Reply:
Worked around by splitting content and tags onto different lines :)